test/e2e/scheduling/OWNERS

```diff
@@ -1,4 +1,5 @@
 approvers:
 - sig-scheduling-maintainers
+- k82cn
```

Comment:
I'm OK with that :). @davidopp , @timothysc do you have other purpose of `sig-scheduling-maintainers`.